catalog/samples/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "Sample App Catalog"
+permalink: /catalog/samples/
+---
+
+Complete applications that demonstrate how to get things done with Flutter. Each sample app features a few classes or an animation, a layout, or other feature of Flutter. The samples are short, just one file and usually only one or two pages of code. They should easy to try out with your favorite IDE.
+
+<div class="container-fluid">
+  <div class="row" style="margin-bottom: 32px">
+    <a href="/catalog/samples/animated-list/">
+      <div class="col-md-3">
+        <img style="border:1px solid #000000" src="https://storage.googleapis.com/flutter-catalog/cb4a54db8fb3726bf4293b9cc5cb12ce16883803/animated_list_small.png" alt="Android screenshot" class="img-responsive">
+      </div>
+   </a>
+    <div class="col-md-9">
+      <p>
+        An AnimatedList that displays a list of cards which stay
+in sync with an app-specific ListModel. When an item is added to or removed
+from the model, the corresponding card animates in or out of view.
```

Comment:
I've made most of these changes, will try and satisfy more wishes in future PRs.